src/components/Request/NewRequestForm.tsx

```diff
@@ -274,6 +327,44 @@ const NewRequestForm = () => {
             <Incentives name="incentives" form={myForm} />
 
             <Telework name="telework" form={myForm} />
+
```

Comment:
BAC-58 AC4 states there should be a notice about the posting being active for 30 days.  It also states there should be a prepopulated unselectable date -- but guessing showing an actual date would be confusing, as I assume it is 30 days from posting -- not necessarily from their submission of the form itself.